src/Apps/App.php

```diff
@@ -80,6 +83,13 @@ public function enableClientMessages(bool $enabled = true)
         return $this;
     }
 
+    public function setCapacity($capacity)
```

Comment:
The other properties have type hints, might as well put one here too.
```suggestion
    public function setCapacity(int $capacity)
```

Reply:
@francislavoie Good note, thank you!.